packages/frontend-core/src/api/logs.js

```diff
@@ -0,0 +1,14 @@
+export const buildLogsEndpoints = API => ({
+  /**
+   * Gets a list of datasources.
+   */
+  getServerLogs: async () => {
+    return await API.get({
+      url: "/api/global/system/logs",
```

Comment:
I think the `/global/system` here could be confusing, as we have currently:
- `/api/global` - tenant level endpoints
- `/api/system` - installation level endpoints

given that the logs are an installation level concept, we could probably just remove `global` here

Reply:
They are system logs. Maybe they should be `api/global/systemlogs` or just `api/global/logs` (I am worried that this can be confusing with the auditlogs or other logs)

Reply:
And we log all the installation, I guess it make sense that it's kept under system

Reply:
My point is that it's potentially confusing to have /api/global/system, when "/api/system" is already a term we use to represent non-tenancy related endpoints. Everything under "/api/global" is supposed to be linked to a tenant. I think "/api/system/logs" makes the most sense? Given were it returns all unfiltered logs across all tenants (if tenancy was enabled)